src/app/components/courses-dashboard/shared/pipes/dateStyle/date.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'dateStyle'
+})
+export class DateStylePipe implements PipeTransform {
+  private monthNames = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec'
+  ];
+
+  transform(date: any): any {
```

Comment:
try to avoid `any` as much as possible.
`transform(date: string): string` - for example.